public-ui/src/app/data-browser/views/ehr-view/components/concept-row-react.component.tsx

```diff
@@ -0,0 +1,388 @@
+        alertEventPageX: e.pageX,
+        alertEventPageY: e.pageY
+      });
+      setTimeout(() => {
+        this.setState({showCopyAlert: false,
+            alertEventPageY: 0,
+            alertEventPageX: 0});
+      }, 400);
+    }
+
+    render() {
+        const {concept, domain, totalResults, maxResults, currentPage, counter, searchTerm, synonymString} = this.props;
+        const {showMoreSynonyms, showMoreDrugBrands, showConceptChart, graphToShow, showCopyAlert,
+        alertEventPageX, alertEventPageY} = this.state;
+        const id = 'c' + concept.conceptId;
+        const tabIndex = 0;
+        const alertBoxTop = alertEventPageY + 10;
+        const  alertBoxTopStr = alertBoxTop + 'px';
+        const alertBoxLeft = alertEventPageX - 60;
+        const  alertBoxLeftStr = alertBoxLeft + 'px';
+        const tooltipAction = 'Concept synonyms tooltip hover on concept ' + concept.conceptName;
+        let conceptIndex = counter + 1;
+        if (totalResults > maxResults) {
+            conceptIndex = counter + ((currentPage - 1) * maxResults) + 1;
+        }
+        const synonymsStr = showMoreSynonyms ? synonymString : (synonymString ? synonymString.substring(0, 100) : null);
+        const drugBrandsStr = showMoreDrugBrands ? concept.drugBrands.join(', ') : concept.drugBrands.slice(0, 10).join(', ');
+        const tblClass = domain === 'labs & measurements' ? 'tbl-r-labs' : 'tbl-r';
+        return <React.Fragment>
+               <style>{cssStyles}</style>
+               <div id={id}>
+                 <div className='tbl-exp-r' onClick={() => this.expandRow()}>
+                 <div className={tblClass} style={domain === 'labs & measurements' ? styles.tblRLabs : styles.tblRow}>
+                 <div className='body-lead tbl-d' style={styles.bodyLead}>
+                    <span>{conceptIndex}. </span>
+                    <HighlightReactComponent searchTerm={searchTerm} text={concept.conceptName}/>
+                 </div>
+                 <div className='body-lead tbl-d' style={styles.bodyLead}>
+                    {concept.countValue <= 20 && <span>&le; </span>} {concept.countValue.toLocaleString()}
+                 </div>
+                 <div className='body-lead tbl-d' style={styles.bodyLead}>
+                    {this.participantPercentage(concept.countValue)} %
+                 </div>
+                 {domain === 'labs & measurements' && concept.measurementConceptInfo &&
+                 <div className='body-lead tbl-d'>
+                    {concept.measurementConceptInfo.hasValues === 1 ?
+                    <span className='test-span' style={styles.measurementTypeSpan}><i className='fas fa-vial' style={{'transform': 'rotate(315deg)'}}></i>
+                    <TooltipReactComponent tooltipKey='valueFilter' label='' searchTerm='' action='' /></span> :
+                    <span className='order-span' style={styles.measurementTypeSpan}><i className='far fa-file-signature'></i>
```

Comment:
Can you use the self-closing syntax on the `<i>` tags here also?

Reply:
Sorry outlooked this.